### DISCUSSION TITLE
right sidebar does not scroll when content is big

### DISCUSSION THREAD

**Discussion by chen19901225** [pradyunsg/furo#831] [Potential Issue]
repo https://github.com/chen19901225/sphinx-example

when click `第106回__薄幸怨檀郎_往事已如烟如雾_温柔怜玉女_伊人真宜喜宜嗔` the page scroll to the content,but the right sidebar 

does not scroll to menu.

**Comment by nineteendo** [comment]
It would be beneficial to link to an online example.
